packages/core/src/external/commonwell/document/document-downloader-local.ts

```diff
@@ -215,7 +242,7 @@ export class DocumentDownloaderLocal extends DocumentDownloader {
           Bucket: s3FileLocation,
           Key: s3FileName,
           Body: pass,
-          ContentType: contentType ? contentType : "text/xml",
+          ContentType: contentType ? contentType : "",
```

Comment:
default will be no contentType which s3 interprets as a binary.

Reply:
in this case, we can do ` ContentType: contentType` as the property is optional - no conditional needed

Reply:
@Goncharo Because we have `exactOptionalPropertyTypes` set in the tsconfig.json, we cant assign to an optional property an undefined. 
I could either change that in tsconfig or just leave as is or do something simpler like this: 
`ContentType: contentType || ""`

Reply:
Changing `tsconfig` shouldn't be an option. The option would be to change the type we return from this function - and likely more upstream - which would be ideal.

We could have a type for the part of the code that can live with a file w/o content type, until the new code you built. From there on, the type would require content type.

Reply:
I am going to stick with:
`ContentType: contentType || "",
`
for simplicities sake